src/web-worker/amp-worker.js

```diff
@@ -29,6 +29,38 @@ const TAG = 'web-worker';
 let PendingMessageDef;
 
 /**
+ * Error that can be passed through web worker
+ * @typedef {{
+ *   message: string,
+ *   stack: string,
+ * }}
+ */
+export let AmpWorkerErrorDef;
```

Comment:
How does this create a dependency on bind-evaluator?

Also, what happens if/when more functionality depends on the worker. This prevents everyone who uses the worker from rolling their own ierror type.